README.md

```diff
@@ -109,9 +109,15 @@ will need to set it up with these steps:
 Check [CONTRIBUTING.md](https://github.com/chshersh/iris/blob/main/CONTRIBUTING.md)
 for contributing guidelines.
 
-To build the project and run the tests, use `cabal`:
+To build the project and run the tests, you can use `cabal`:
 
 ```shell
 cabal build all
 cabal test --enable-tests --test-show-details=direct
 ```
+or `stack`:
+
+```shell
+stack build
+stack test
+```
```

Comment:
That's great! I'm thinking about making instructions more beginner-friendly so everyone without prior knowledge of Haskell tooling can have a quick start guide or at least pointers to where get the tooling. So something like this:

```markdown
To build the project and run the tests locally, you can use either `cabal` or `stack`.

### Cabal

1. Install [GHCup](...). It also install GHC (a Haskell compiler) and `cabal` (a Haskell build tool).
2. Build the project
3. Run the tests

### Stack

1. Install [Stack](...)
2. Build the project
3. Run the tests
```